honeybee_energy/load/_base.py

```diff
@@ -0,0 +1,78 @@
+# coding=utf-8
+"""Base object for all load definitions."""
+from __future__ import division
+
+from ..schedule.ruleset import ScheduleRuleset
+from ..schedule.fixedinterval import ScheduleFixedInterval
+
+from honeybee._lockable import lockable
+from honeybee.typing import valid_ep_string
+
+
+@lockable
+class _LoadBase(object):
+    """A base object for all load definitions.
+
+    Properties:
+        * name
```

Comment:
I realized that this is what I should do in order to get bullet lists to display correctly in the sphinx docs.   Otherwise, we get something that looks like [what you see under properties here](https://www.ladybug.tools/honeybee-energy/docs/honeybee_energy.construction.html). Of course, this is a hidden module so it doesn't really matter too much here but I'm trying to follow this as a good practice convention across all my docstrings.

Reply:
I see. Nice. Another item to add to the issue that you have for documentation.